src/api/app/views/webui/request/_delete_request_dialog.html.haml

```diff
@@ -0,0 +1,21 @@
+.dialog.darkgrey_box
+  .box.box-shadow
+    %h2.box-header Create Delete Request
+    .dialog-content
+      %p
+        Do you really want to request the deletion of #{project_or_package_link project: @project, package: @package}?
+      %p.smaller
+        = "You are about to send a request to the owner of the original #{ @package ? 'package' : 'project' }. Please, describe your reasons to delete this #{ @package ? 'package' : 'project' }. The owner will be notified with the following details."
```

Comment:
I think it would be even better with `Please describe your reasons to request the deletion of (...)`. Because this is only a request.